lib/src/sqlalchemy/apps/models/query.py

```diff
@@ -0,0 +1,9 @@
+from typing import Optional
+
+from pydantic import Field
+
+from .connection import Connection
+
+
+class Query(Connection):
```

Comment:
The description is fetched to the interface. If can be more verbose, than better to write more.